client/state/billing-transactions/test/actions.js

```diff
@@ -15,6 +15,8 @@ describe( 'actions', () => {
 		spy = jest.fn();
 	} );
 
+	const EndpointLimit = 200;
```

Comment:
Super minor (especially since this is only a test) but this should probably have a lowercase first letter because it's not a class or a React component which typically are the only things that get capitalized in JS. `const` is just a regular variable definition.

```suggestion
	const endpointLimit = 200;
```

Reply:
Dang. I saw it and forgot to rectify. Thanks!